webpack.config.renderer.dev.js

```diff
@@ -234,6 +237,31 @@ export default merge.smart(baseConfig, {
 
     new ExtractTextPlugin({
       filename: '[name].css'
+    }),
+
+    new HtmlWebpackPlugin({
+      template: path.resolve(__dirname, 'app', 'app.html')
+    }),
+
+    new AddAssetHtmlPlugin({
+      filepath: path.resolve(__dirname, 'dll', 'renderer.dev.dll.js'),
```

Comment:
I think we want to use `path.join` here, as `path.resolve` has functionality we're not relying on: https://nodejs.org/docs/latest/api/path.html#path_path_resolve_paths